linux-tools/sqlite/sqlite.sh

```diff
@@ -29,9 +29,10 @@
 ## source the utility functions
 
 #cd `dirname $0`
-#LTPBIN=${LTPBIN%/shared}/sqlite
+LTPBIN="/usr/local/autotest_base/autotest-client-tests/linux-tools/shared"
 source $LTPBIN/tc_utils.source
```

Comment:
This may not work if tests are placed under different folder. Any other logic can be used here ?